connectors/src/connectors/zendesk/lib/help_center_permissions.ts

```diff
@@ -159,6 +151,39 @@ export async function allowSyncZendeskCategory({
     const zendeskApiClient = createZendeskClient(
       await getZendeskSubdomainAndAccessToken(connectionId)
     );
+
+    /// creating the brand if missing
+    const brand = await ZendeskBrandResource.fetchByBrandId({
```

Comment:
The brand can not exist when we process the first category if the first permission given is at the category level?

Reply:
yes, in `retrievePermissions` with `permissions == none` we can select the category even when the brand has never been selected

Reply:
happy to revalidate that later!